docs/en/observability/monitor-k8s/monitor-k8s-overview.asciidoc

```diff
@@ -0,0 +1,89 @@
+[discrete]
+[[kubernetes-monitoring-architecture]]
+== Monitoring architecture
+
+The {stack} provides 4 main components for monitoring Kubernetes:
+
+1. Lightweight agents, called {beats}, to collect observability data. Some
+{beats} include pre-configured data collection modules to ease the collection
+and parsing of data for common applications such as Apache, MySQL, and Kafka.
+
+2. APM (described later) to monitor, detect, and diagnose complex application
+performance issues.
+
+3. {es} for storing and searching your data.
+
+4. Observability apps in {kib} for visualizing and managing your observability
+data.
+
+//REVIEWERS: I just threw some icons into this diagram for services that
```

Comment:
I think the only one that is not so related is the `docker` icon. Docker is not a user's application. It is a core component on each of the hosts. I would avoid adding this.

Redis, Nginx, Apache, Django, NodeJS, Kafka, Etcd, Mongo etc, are kind of workloads I expect to see in such an example.

Reply:
Right...I meant it be generic (you have some other docker container running in this pod), but I see where that might be confusing. :-)